### PR TITLE
Add pyxid

### PR DESCRIPTION
**Pull request by kastman**

### CODE REVIEW

File: recipes/pyxid/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    - setuptools
+    - pyserial >=2.5
+
+
+  run:
+    - python
+    - pyserial >=2.5
+
+test:
+  # Python imports
```
Comment (jakirkham) [inline recipes/pyxid/meta.yaml]: This comment can be dropped.

File: recipes/pyxid/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  imports:
+    - pyxid
+
+about:
+  home: http://www.github.com/cedrus-opensource/pyxid/
+  license: BSD
```
Comment (jakirkham) [inline recipes/pyxid/meta.yaml]: What version of BSD (e.g. BSD 3-Clause)? Also is there a license file that we can include in the package from the source?

File: recipes/pyxid/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  summary: 'Pure python library for communicating with Cedrus XID and StimTracker devices.'
+  license_family: BSD
+  author: "Grant Limberg"
+  author_email: "developers@cedrus.com"
+  maintainer: "Cedrus Corporation"
+  maintainer_email: "opensource@cedrus.com"
```
Comment (jakirkham) [inline recipes/pyxid/meta.yaml]: This is new. @msarahan is this metadata supported by `conda-build`?

File: recipes/pyxid/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{% set name = "pyxid" %}
+{% set version = "1.0" %}
+{% set sha256 = "e689bce2ed8ed4bff7174a3a79f8b639ebf6356807aec7e9da481217313214ff" %}
+
+package:
+  name: pyxid
+  version: "1.0"
```
Comment (jakirkham) [inline recipes/pyxid/meta.yaml]: Please replace with `version: "{{ version }}"`. Quotes are optional.

File: recipes/pyxid/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyserial >=2.5
+
+
```
Comment (jakirkham) [inline recipes/pyxid/meta.yaml]: nit: Please only have one blank line between `build` and `run`.

File: recipes/pyxid/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+  home: http://www.github.com/cedrus-opensource/pyxid/
+  license: BSD
+  summary: 'Pure python library for communicating with Cedrus XID and StimTracker devices.'
+  license_family: BSD
+  author: "Grant Limberg"
+  author_email: "developers@cedrus.com"
+  maintainer: "Cedrus Corporation"
+  maintainer_email: "opensource@cedrus.com"
+
+
```
Comment (jakirkham) [inline recipes/pyxid/meta.yaml]: nit: Please only have one blank line between `about` and `extra`.

File: recipes/pyxid/meta.yaml (hunk 3 above)
Comment (kastman) [inline recipes/pyxid/meta.yaml]: Sorry, I just grabbed these all from the ``setup.py``; I've taken them back out. Extra keys in the yaml don't break ``conda-build``, but I think it's cleaner not to have them.

File: recipes/pyxid/meta.yaml [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: true  # [py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyserial >=2.5
```
Comment (jakirkham) [inline recipes/pyxid/meta.yaml]: Is `pyserial` needed at build time? If not, could we please remove it?

**Comment by jakirkham** [comment]
One final change then LGTM.

File: recipes/pyxid/meta.yaml (hunk 7 above)
Comment (kastman) [inline recipes/pyxid/meta.yaml]: Sorry for the delay! Yes, `pyserial` is needed at build time (dependency failure without it), so we need to leave it in.

```
Installed /Users/kastman/miniconda/conda-bld/pyxid_1488744704259/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python2.7/site-packages/pyxid-1.0-py2.7.egg
Processing dependencies for pyxid==1.0
Searching for pyserial>=2.5
Traceback (most recent call last):
  File "setup.py", line 26, in <module>
    "Operating System :: POSIX",
  File "/Users/kastman/miniconda/conda-bld/pyxid_1488744704259/_b_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pla/lib/python2.7/distutils/core.py", line 151, in setup
...
...
```

**Comment by jakirkham** [comment]
Thanks @kastman.

**Pull request merged**